src/index.js

```diff
@@ -215,6 +215,7 @@ class UsersRouting extends React.Component {
               <Route path={`${base}/:id/patronblocks/create`} component={Routes.PatronBlockContainer} />
               <Route path={`${base}/create`} component={Routes.UserEditContainer} />
               <Route path={`${base}/lost-items`} component={Routes.LostItemsContainer} />
+              <Route path={`${base}/pre-registration-records`} component={Routes.PatronPreRegistrationRecordsContainer} />
```

Comment:
You protect the action but not the route

https://github.com/folio-org/ui-users/pull/2755/files#diff-801062542feb64887040b0917e0cb3fcc76055e2c6a0c979c0694121f21001d8R15

Reply:
Yes, instead the search button on the new route is disabled in the absence of permission. This is agreed by our PO.